tests/e2e/support/coingecko-stubs.js

```diff
@@ -0,0 +1,203 @@
+beforeEach(() => {
```

Comment:
It was done here https://github.com/superhero-com/superhero-wallet/commit/aa852fa649a0a13deffb08038677492b413f8ddf .
Isn't it working properly?